packages/angular/src/schematics/library/library.ts

```diff
@@ -406,9 +406,12 @@ function updateTsConfig(options: NormalizedSchema): Rule {
       return updateJsonInTree('tsconfig.json', json => {
         const c = json.compilerOptions;
         delete c.paths[options.name];
-        c.paths[`@${nxJson.npmScope}/${options.projectDirectory}`] = [
-          `libs/${options.projectDirectory}/src/index.ts`
-        ];
+
+        const pathName = options.publishable
+          ? `@${nxJson.npmScope}/${options.name}`
+          : `@${nxJson.npmScope}/${options.projectDirectory}`;
+
+        c.paths[pathName] = [`libs/${options.projectDirectory}/src/index.ts`];
```

Comment:
hmm 

your changes do following right?

1. For `nx g @nrwl/react:lib --name=core --directory=platform --publishable`
- create path mapping `{ "@my-org/core": "libs/platform/core/src/index.ts" }` 

👉 This should be rather `{ "@my-org/platform-core": "libs/platform/core/src/index.ts" }`   no?


2. For `nx g @nrwl/react:lib --name=core --publishable`
- create path mapping `{ "@my-org/core": "libs/platform/core/src/index.ts" }`

I know it's a draft PR :D , sorry to chime in

Reply:
Hey, no, it's fine 🙂. Feel free to chime in at any point. We didn't move ahead here with this as we're currently discussing internally how in general buildable vs publishable should be handled. Currently they're kind of synonyms but they don't necessarily have to. You could have a buildable library but not want to publish it etc.

So atm this is on hold and we'll pick it up afterwards